bundle/language_ar.properties

```diff
@@ -1,1505 +1,1493 @@
+angal.common.excel.btn                                                           = Excel
+angal.common.excel.btn.key                                                       = E
+angal.common.female.btn                                                          = Female
+angal.common.filter.btn                                                          = Filter
+angal.common.filter.btn.key                                                      = F
+angal.common.launchreport.btn                                                    = Launch Report 
+angal.common.launchreport.btn.key                                                = R
+angal.common.male.btn                                                            = Male
+angal.common.new.btn                                                             = جديد
+angal.common.new.btn.key                                                         = N
+angal.common.newpatient.btn                                                      = New Patient
+angal.common.newpatient.btn.key                                                  = N
+angal.common.ok.btn                                                              = OK
+angal.common.ok.btn.key                                                          = O
+angal.common.print.btn                                                           = Print
+angal.common.print.btn.key                                                       = P
+angal.common.save.btn                                                            = حفظ
+angal.common.save.btn.key                                                        = S
+angal.common.search.btn                                                          = Search
+angal.common.search.btn.key                                                      = S
+angal.common.stockcard.btn                                                       = StockCard
+angal.common.stockcard.btn.key                                                   = K
+angal.common.stockledger.btn                                                     = Stock Ledger
+angal.common.stockledger.btn.key                                                 = L
+angal.common.submit.btn                                                          = Submit
+angal.common.submit.btn.key                                                      = S
+angal.common.select.btn                                                          = Select
+angal.common.select.btn.key                                                      = S
+angal.common.update.btn                                                          = Update
+angal.common.update.btn.key                                                      = U
+
+###########################
+# Common messages and text
+###########################
+angal.common.pleaseselectarow.msg                                                = يرجى اختيار صف
+angal.common.datacouldnotbesaved.msg                                             = The data could not be saved.
```

Comment:
Same doubts as in https://github.com/informatici/openhospital-gui/pull/585#discussion_r658070144